### PR TITLE
ios-deploy 1.11.1

### PR DESCRIPTION
**Pull request by gabebear**
Manually created because of getting error when running locally.

```
brew bump-formula-pr --url="https://github.com/ios-control/ios-deploy/archive/1.11.1.tar.gz" ios-deploy
```
gives the error
```
Error: You need to bump this formula manually since the new URL
and old URL are both:
  https://github.com/ios-control/ios-deploy/archive/1.11.1.tar.gz
```

### CODE REVIEW

**Comment by gabebear** [comment]
ios-deploy is mostly GPL-3.0-or-later with a single file in BSD-3-Clause-Attribution

https://github.com/ios-control/ios-deploy#license

**Comment by Rylan12** [comment]
> ios-deploy is mostly GPL-3.0-or-later with a single file in BSD-3-Clause-Attribution
> 
> https://github.com/ios-control/ios-deploy#license

If that's the case, I think we still need to list both `GPL-3.0-or-later` and `BSD 3-Clause`. [The PR](https://github.com/Homebrew/brew/pull/8260) to enable support for this has been merged, but we need to wait for a new Homebrew release before we can start including these in formulae.

**Comment by gabebear** [comment]
I went ahead and added an "all_of" for the two licenses used.

How long until a new release goes out?

**Comment by dawidd6** [comment]
Squash your commits first

**Comment by gabebear** [comment]
I can squash them and rewrite the history on the remote beach later... does your process require this?

If you don’t have access to the “Confirm Squash and Merge” button it might be good to contact an owner of the Homebrew project to get it enabled. https://github.blog/2016-04-01-squash-your-commits/

**Comment by chenrui333** [comment]
> I can squash them and rewrite the history on the remote beach later... does your process require this?

yes, we require that to make the commit messages conforming with our standard.

> If you don’t have access to the “Confirm Squash and Merge” button it might be good to contact an owner of the Homebrew project to get it enabled. https://github.blog/2016-04-01-squash-your-commits/

We are using that every day for formulae which do not need bottles. And it does not apply in here.

**Comment by chenrui333** [comment]
> Manually created because of getting error when running locally.
> 
> ```
> brew bump-formula-pr --url="https://github.com/ios-control/ios-deploy/archive/1.11.1.tar.gz" ios-deploy
> ```
> 
> gives the error
> 
> ```
> Error: You need to bump this formula manually since the new URL
> and old URL are both:
>   https://github.com/ios-control/ios-deploy/archive/1.11.1.tar.gz
> ```

That normally happens when you already updated the version locally, and `bump-formula-pr` dont know what to do. You should be able to tell based on `git diff`

**Comment by chenrui333** [comment]
just rebased your commits, and you are welcome.

**Comment by Rylan12** [comment]
I'm not sure when the next release will come out. I think they're usually done on Mondays but don't quote me on that. The best thing to do is probably to remove the license line and replace it with a comment so we know what to replace it with. We can then re-add the license in a future PR once there's been a new release. Sorry, I should have specified that earlier.

We can't use the Squash and Merge button for most PRs because we need to build binary bottles as part of the merge process. Certain formulae don't need these bottles, so we can use the GitHub UI to merge. In this case, though, we do need to update the bottles. A maintainer can do this manually, but it will usually be handled automatically by a GitHub action workflow.

**Comment by chenrui333** [comment]
Done with commenting out the license update.

**Review by Rylan12 — APPROVED** [review]

**Comment by Rylan12** [comment]
LGTM once CI is green.

Thanks, @gabebear and @chenrui333!

**Pull request closed without merging**
